apps/src/templates/YourSchoolResources.jsx

```diff
@@ -0,0 +1,73 @@
+import React, {Component} from 'react';
+import Radium from 'radium';
+import ContentContainer from './ContentContainer';
+import Button from './Button';
+import styleConstants from '../styleConstants';
+import i18n from "@cdo/locale";
+import _ from 'lodash';
+import {pegasus} from '@cdo/apps/lib/util/urlHelpers';
+
+const contentWidth = styleConstants['content-width'];
+
+const styles = {
+  container: {
+    width: contentWidth,
+    display: "flex",
+    justifyContent: "space-between"
+  },
+  regularRow: {
+    marginBottom: 20
+  }
+};
+
+class YourSchoolResources extends Component {
+
+  buttons = [
+    {
+      text: i18n.teachers(),
+      path: 'educate'
+    },
+    {
+      text: i18n.administrators(),
+      path: 'educate/district'
+    },
+    {
+      text: i18n.parents(),
+      path: 'promote/letter'
+    },
+  ];
+
+  render() {
+    return (
+      <ContentContainer isRtl={false}>
+        {_.chunk(this.buttons, 3).map(
```

Comment:
Are we expecting more buttons in the future? Will there by a dynamic number of them? Wondering if the chunking is actually necessary here.